.gitattributes

```diff
@@ -0,0 +1,2 @@
+# https://prettier.io/docs/en/options.html#end-of-line
+* text=auto eol=lf
```

Comment:
Enforces LF EOL when doing a checkout of this project. On Windows, `git commit` fails when `core.autocrlf: true` because of the pre-commit hook running prettier, which expects file with LF EOL. Let me know if you prefer to not add this file.

Reply:
nice one